docs/managing-your-workflow.md

```diff
@@ -0,0 +1,101 @@
+# Managing your workflow
+[Prepare](#Prepare-your-workflow) your workflow <br/>
```

Comment:
is it possible to make these clickable links that go to the actual  section below

Reply:
they are clickable links that  jump to sections. For the best UX, go to the main page of the GitHub repo and change the branch to jbagga/docsRefactor. All links should work. This view doesn't have that enabled